app/(student)/learning/[slug]/lecture/[lectureId]/page.tsx

```
@@ -1,286 +1,333 @@
-                                className="px-4 py-2 bg-blue-600 text-white rounded-md hover:bg-blue-700"
-                            >
-                                Nộp bài tập
-                            </button>
-                        </div>
-                    </div>
-                </div>
+          <div className="p-6 prose max-w-none">
+            <div
+              dangerouslySetInnerHTML={{
+                __html: assignment.instructions
+                  .replace(
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)